karspexet/ticket/templates/economy/show_detail.html

```diff
@@ -0,0 +1,50 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% block content %}
+<h2>Föreställningsöversikt: {{show.production.name}} {{show.date|date:"Y-m-d H:i:s" }}</h2>
+<em>{{show.venue}}</em>
+
+<h3>Inloggad som: {{ user }}</h3>
+
+<h3>Sålda biljetter</h3>
+
+<dl>
+  <dt>Antal sålda biljetter</dt>
+  <dd>{{ coverage.ticket_count }}</dd>
+  <dt>Antal platser</dt>
+  <dd>{{ coverage.seat_count }}</dd>
+  <dt>Beläggning</dt>
+  <dd>{{ coverage.sales_percentage|floatformat }} %</dd>
+</dl>
+
+<span class="theater-seat-map">
+    {% include "seat_map.svg" with venue=show.venue taken_seats=taken_seats %}
+</span>
+<link rel="stylesheet" href="{% static 'select_seats.css' %}">
+
+
+<table>
+  <thead>
+    <tr>
+      <th>Sektion</th>
+      <th>Plats</th>
+      <th>Namn</th>
+      <th>Telefonnummer</th>
+      <th>E-postadress</th>
+      <th>Biljettyp</th>
+    </tr>
+  </thead>
+  <tbody>
+    {% for ticket in show.ticket_set.all %}
```

Comment:
You don't need parentheses after `all´?

_Pun intended_

Reply:
Hah, true.

Reply:
You don't need parentheses when you do this in a template. Maybe when using jinja2, but we're not.